mllib/src/main/scala/org/apache/spark/ml/classification/ProbabilisticClassifier.scala

```diff
@@ -205,7 +205,11 @@ private[spark] abstract class ProbabilisticClassificationModel[
   }
 }
 
-private[ml] object ProbabilisticClassificationModel {
+/**
+ * :: DeveloperApi ::
+ */
+@DeveloperApi
+object ProbabilisticClassificationModel {
```

Comment:
Do we want to make it public? It exposes the `normalizeToProbabilitiesInPlace` method.